frontend/test/__runner__/backend.js

```diff
@@ -70,15 +70,6 @@ const BackendResource = createSharedResource("BackendResource", {
                 process.env["ENTERPRISE_TOKEN"]) ||
               undefined,
             MB_FIELD_FILTER_OPERATORS_ENABLED: "true",
```

Comment:
@daltojohnso - we don't need this "flag" anymore, right? This is now an official part of Metabase.